features-json/webgpu.json

```diff
@@ -657,7 +657,6 @@
       "3.0-3.1":"n d #1"
     }
   },
-  "notes":"All major browser engines are working on implementing this spec.",
```

Comment:
I believe it would need to be

```
"notes":"",
```

to make the pipeline happy :)

(sadly I can't add it as a proper easy to apply suggestion)